amazon-dynamodb/1.4/modules/ROOT/pages/amazon-dynamodb-connector-reference.adoc

```diff
@@ -697,26 +697,26 @@ For information, see the http://docs.aws.amazon.com/amazondynamodb/latest/APIRef
 ** ALL_PROJECTED_ATTRIBUTES
 ** SPECIFIC_ATTRIBUTES
 ** COUNT |  The attributes to be returned in the result. |  |
-| Attribute Names a| Object |  |  |
-| Attribute Values a| Object |  |  |
-| Table Name a| String |  |  | x
-| Attributes To Gets a| Array of String |  |  |
+| Attribute Names a| Object | One or more substitution tokens for attribute names in an expression. |  |
+| Attribute Values a| Object | One or more values to substitute in an expression. |  |
+| Table Name a| String | Name of the table containing the requested item. |  | x
+| Attributes To Gets a| Array of String | This is a legacy parameter. Use the Projection Expression parameter instead. |  |
 | Conditional Operator a| Enumeration, one of:
 
 ** AND
-** OR |  |  |
-| Consistent Read a| Boolean |  |  false |
-| Exclusive Start Key a| Object |  |  |
-| Filter Expression a| String |  |  |
-| Index Name a| String |  |  |
-| Limit a| Number |  |  |
-| Projection Expression a| String |  |  |
-| Condition Filter a| Object |  |  |
+** OR | A logical operator to apply to the conditions. |  |
+| Consistent Read a| Boolean | Determines the read consistency model: If set to true, then the operation uses strongly consistent reads; otherwise, the operation uses eventually consistent reads. |  false |
+| Exclusive Start Key a| Object | The primary key of the first item that this operation evaluates. |  |
+| Filter Expression a| String | A string that contains conditions that DynamoDB applies after the operation, but before the data is returned to you. Items that do not satisfy the criteria are not returned. |  |
+| Index Name a| String | The name of a secondary index to scan. This index can be any local secondary index or global secondary index. |  |
+| Limit a| Number | The maximum number of items to evaluate (not necessarily the number of matching items).  |  |
+| Projection Expression a| String | A string that identifies one or more attributes to retrieve from the table. |  |
+| Condition Filter a| Object | The condition that evaluates the query results after the items are read and returns only the desired values. |  |
 | Return Consumed Capacity a| Enumeration, one of:
 
 ** INDEXES
 ** TOTAL
-** NONE |  |  |
+** NONE | Determines the level of detail about provisioned throughput consumption returned in the response. |  |
```

Comment:
For consistency with other descriptions, remove the verb (and another wording suggestion):

The level of detail of the provisioned throughput consumption returned in the response.

Ditto for other occurrences of this field.